source/language/gates.rst

```diff
@@ -167,13 +167,13 @@ and applies the controlled gate
 
 .. _sec:macros:
 
-Hierarchically defined unitary gates
-------------------------------------
+User-defined unitary gates
+--------------------------
 
-For new gates, we associate them with a corresponding unitary
-transformation by a sequence of built-in gates. For example, a CPHASE
-operation is shown schematically in :numref:`fig_gate`
-corresponding OpenQASM code is
+Programmers may define new gates, which may be resolved to a sequence of built-in
```

Comment:
Nit: `which can be resolved` - it's required not optional that resolution exists to a sequence of built-in gates.

Reply:
Noted and amended